met-web/src/services/userService/api/index.tsx

```diff
@@ -23,9 +23,10 @@ export const getUserList = async (params: GetUserParams = {}): Promise<Page<User
 interface AddUserToGroupProps {
     user_id?: string;
     group?: string;
+    engagement_id?: number;
 }
-export const addUserToGroup = async ({ user_id, group }: AddUserToGroupProps): Promise<User> => {
+export const addUserToGroup = async ({ user_id, group, engagement_id }: AddUserToGroupProps): Promise<User> => {
```

Comment:
i think add user to group API is different from add user as a member to an engagement ; so not sure we can put it in common place...
The member is an engagement level api `api/engagements/{eng_id}/members`

Reply:
Done